core/arch/arm/plat-hikey/platform_flags.mk

```diff
@@ -29,3 +29,5 @@ user_ta-platform-cflags += $(arm32-platform-cflags)
 user_ta-platform-cflags += -fpie
 user_ta-platform-cppflags += $(arm32-platform-cppflags)
 user_ta-platform-aflags += $(arm32-platform-aflags)
+
+CFG_HIKEY_UART3 ?= n
```

Comment:
Ok thanks. Will remove. For future references, what would be a good place to put platform specific definitions like this?